reference/tools/meson/mesontoolchain.rst

```diff
@@ -284,6 +284,25 @@ in this example of host profile:
     tools.apple:sdk_path=/my/path/to/iPhoneOS.sdk
 
 
+**Android**
+
+It initializes the ``MesonToolchain`` ``c``, ``cpp``, and ``ar`` attributes, which are needed to cross-compile for Android, given the
+Conan settings for Android and the ``tools.android:ndk_path`` configuration value like it's shown
+in this example of host profile:
+
+
+.. code-block:: text
+    :caption: **android_host_profile**
+
+    [settings]
+    os = Android
+    os.api_level = 21
+    arch = armv8
+
+    [conf]
+    tools.android:ndk_path=/my/path/to/NDK
+
+
```

Comment:
This was in the wrong section because of conflicts merging the branches I guess.